epytope/EpitopePrediction/External.py

```diff
@@ -5853,6 +5985,1200 @@ def parse_external_result(self, file):
+'HLA-DQA1*06:02-DQB1*02:05', 'HLA-DQA1*06:02-DQB1*02:06', 'HLA-DQA1*06:02-DQB1*03:01', 'HLA-DQA1*06:02-DQB1*03:02', 'HLA-DQA1*06:02-DQB1*03:03',
+'HLA-DQA1*06:02-DQB1*03:04', 'HLA-DQA1*06:02-DQB1*03:05', 'HLA-DQA1*06:02-DQB1*03:06', 'HLA-DQA1*06:02-DQB1*03:07', 'HLA-DQA1*06:02-DQB1*03:08',
+'HLA-DQA1*06:02-DQB1*03:09', 'HLA-DQA1*06:02-DQB1*03:10', 'HLA-DQA1*06:02-DQB1*03:11', 'HLA-DQA1*06:02-DQB1*03:12', 'HLA-DQA1*06:02-DQB1*03:13',
+'HLA-DQA1*06:02-DQB1*03:14', 'HLA-DQA1*06:02-DQB1*03:15', 'HLA-DQA1*06:02-DQB1*03:16', 'HLA-DQA1*06:02-DQB1*03:17', 'HLA-DQA1*06:02-DQB1*03:18',
+'HLA-DQA1*06:02-DQB1*03:19', 'HLA-DQA1*06:02-DQB1*03:20', 'HLA-DQA1*06:02-DQB1*03:21', 'HLA-DQA1*06:02-DQB1*03:22', 'HLA-DQA1*06:02-DQB1*03:23',
+'HLA-DQA1*06:02-DQB1*03:24', 'HLA-DQA1*06:02-DQB1*03:25', 'HLA-DQA1*06:02-DQB1*03:26', 'HLA-DQA1*06:02-DQB1*03:27', 'HLA-DQA1*06:02-DQB1*03:28',
+'HLA-DQA1*06:02-DQB1*03:29', 'HLA-DQA1*06:02-DQB1*03:30', 'HLA-DQA1*06:02-DQB1*03:31', 'HLA-DQA1*06:02-DQB1*03:32', 'HLA-DQA1*06:02-DQB1*03:33',
+'HLA-DQA1*06:02-DQB1*03:34', 'HLA-DQA1*06:02-DQB1*03:35', 'HLA-DQA1*06:02-DQB1*03:36', 'HLA-DQA1*06:02-DQB1*03:37', 'HLA-DQA1*06:02-DQB1*03:38',
+'HLA-DQA1*06:02-DQB1*04:01', 'HLA-DQA1*06:02-DQB1*04:02', 'HLA-DQA1*06:02-DQB1*04:03', 'HLA-DQA1*06:02-DQB1*04:04', 'HLA-DQA1*06:02-DQB1*04:05',
+'HLA-DQA1*06:02-DQB1*04:06', 'HLA-DQA1*06:02-DQB1*04:07', 'HLA-DQA1*06:02-DQB1*04:08', 'HLA-DQA1*06:02-DQB1*05:01', 'HLA-DQA1*06:02-DQB1*05:02',
+'HLA-DQA1*06:02-DQB1*05:03', 'HLA-DQA1*06:02-DQB1*05:05', 'HLA-DQA1*06:02-DQB1*05:06', 'HLA-DQA1*06:02-DQB1*05:07', 'HLA-DQA1*06:02-DQB1*05:08',
+'HLA-DQA1*06:02-DQB1*05:09', 'HLA-DQA1*06:02-DQB1*05:10', 'HLA-DQA1*06:02-DQB1*05:11', 'HLA-DQA1*06:02-DQB1*05:12', 'HLA-DQA1*06:02-DQB1*05:13',
+'HLA-DQA1*06:02-DQB1*05:14', 'HLA-DQA1*06:02-DQB1*06:01', 'HLA-DQA1*06:02-DQB1*06:02', 'HLA-DQA1*06:02-DQB1*06:03', 'HLA-DQA1*06:02-DQB1*06:04',
+'HLA-DQA1*06:02-DQB1*06:07', 'HLA-DQA1*06:02-DQB1*06:08', 'HLA-DQA1*06:02-DQB1*06:09', 'HLA-DQA1*06:02-DQB1*06:10', 'HLA-DQA1*06:02-DQB1*06:11',
+'HLA-DQA1*06:02-DQB1*06:12', 'HLA-DQA1*06:02-DQB1*06:14', 'HLA-DQA1*06:02-DQB1*06:15', 'HLA-DQA1*06:02-DQB1*06:16', 'HLA-DQA1*06:02-DQB1*06:17',
+'HLA-DQA1*06:02-DQB1*06:18', 'HLA-DQA1*06:02-DQB1*06:19', 'HLA-DQA1*06:02-DQB1*06:21', 'HLA-DQA1*06:02-DQB1*06:22', 'HLA-DQA1*06:02-DQB1*06:23',
+'HLA-DQA1*06:02-DQB1*06:24', 'HLA-DQA1*06:02-DQB1*06:25', 'HLA-DQA1*06:02-DQB1*06:27', 'HLA-DQA1*06:02-DQB1*06:28', 'HLA-DQA1*06:02-DQB1*06:29',
+'HLA-DQA1*06:02-DQB1*06:30', 'HLA-DQA1*06:02-DQB1*06:31', 'HLA-DQA1*06:02-DQB1*06:32', 'HLA-DQA1*06:02-DQB1*06:33', 'HLA-DQA1*06:02-DQB1*06:34',
+'HLA-DQA1*06:02-DQB1*06:35', 'HLA-DQA1*06:02-DQB1*06:36', 'HLA-DQA1*06:02-DQB1*06:37', 'HLA-DQA1*06:02-DQB1*06:38', 'HLA-DQA1*06:02-DQB1*06:39',
+'HLA-DQA1*06:02-DQB1*06:40', 'HLA-DQA1*06:02-DQB1*06:41', 'HLA-DQA1*06:02-DQB1*06:42', 'HLA-DQA1*06:02-DQB1*06:43', 'HLA-DQA1*06:02-DQB1*06:44',
+'H-2-IAb', 'H-2-IAd', 'H-2-IAk', 'H-2-IAq', 'H-2-IAs',
```

Comment:
Did you also check if it works with mouse alleles?

Reply:
No I did not good point! As I checked this for all predictors I even found a bug described in #48. I'll fixed that in the next commit

Reply:
No I did not good point! As I checked this for all predictors I even found a bug described in #48. I'll fixed that in the next commit